shell/common/rasterizer.cc

```diff
@@ -787,12 +787,11 @@ DrawSurfaceStatus Rasterizer::DrawToSurfaceUnsafe(
 
     frame->set_submit_info(submit_info);
 
-    if (external_view_embedder_ &&
-        (!raster_thread_merger_ || raster_thread_merger_->IsMerged())) {
+    if (external_view_embedder_) {
```

Comment:
TODO: Clarify rasterizer behavior is actually (if platformViews) and remove usage of raster thread merger as sentinel.

Reply:
Done